starboard/shared/starboard/player/filter/audio_frame_discarder.cc

```diff
@@ -49,7 +49,9 @@ void AudioFrameDiscarder::AdjustForDiscardedDurations(
     scoped_refptr<DecodedAudio>* decoded_audio) {
   SB_DCHECK(decoded_audio);
   SB_DCHECK(*decoded_audio);
-  SB_DCHECK(!input_buffer_infos_.empty());
+  // TODO: Comment out the SB_DCHECK due to b/274021285. We can re-enable it
+  // after b/274021285 is resolved.
+  // SB_DCHECK(!input_buffer_infos_.empty());
 
```

Comment:
Let's add a SB_LOG(WARNING) here, with |input_buffer_infos_.size()| in the message.